### DISCUSSION TITLE
Hyde-hyprpanel

### DISCUSSION THREAD

**Discussion by rubiin** [kRHYME7/Hyde-cli#94] [Q&A] [answered]
Do we need to disable waybar for hyprpanel or the scripts does any toggling

**Comment by kRHYME7** [comment, marked as answer]
There's no script yet. So we need to manually disable waybar

**Reply by rubiin** [reply]
Also we should enable discussions on all repo for disabling cross repo posting

**Reply by rubiin** [reply]
@kRHYME7 `${cacheDir}/landing/wallbash-hyprpanel.json | ags -r "useTheme('${cacheDir}/landing/wallbash-hyprpanel.json')"` is this the command used inplace of waybar
